src/librustdoc/html/static/js/externs.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
+var searchState;
+function initSearch(searchIndex){}
+
+/* eslint-disable */
+
+/**
+ * @typedef {{
+ *   raw: string,
+ *   query: string,
+ *   type: string,
+ *   id: string,
+ * }}
+ */
+var ParsedQuery;
+
+/**
+ * @typedef {{
+ *    crate: string,
+ *    desc: string,
+ *    id: number,
+ *    name: string,
+ *    normalizedName: string,
+ *    parent: (Object|null|undefined),
+ *    path: string,
+ *    ty: (Number|null|number),
+ *    type: (Array<?>|null)
+ * }}
+ */
+var Row;
+
+/* eslint-enable */
```

Comment:
I thought `/* eslint-disable */` was only for the current file. Oh well. Strange haha.